src/app/acesso/cadastro/info-pessoal-cadastro/info-pessoal-cadastro.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit, Input, ChangeDetectorRef, ChangeDetectionStrategy, Output, EventEmitter } from '@angular/core';
+import { Usuario } from 'src/app/interfaces/usuario';
+import { FormControl, Validators } from '@angular/forms';
+import {telefoneFormControl, cpfFormControl, nomeFormControl, emailFormControl, senhaFormControl} from '../../../util/form-controls';
```

Comment:
Codacy found an issue: [A space is required after '{'.](https://app.codacy.com/manual/gaugsouza/petshow-frontend/pullRequest?prid=6334730)